packages/plugin-phone/test/integration/spec/media-state-controls.js

```diff
@@ -144,8 +145,9 @@ describe(`plugin-phone`, function() {
         });
 
         describe(`when the call is not receiving audio`, () => {
```

Comment:
Before eliminating the bugzilla comment, please run this test in firefox via sauce. I think you'll find that it fails with a different error